components/Aside.vue

```diff
@@ -1,10 +1,26 @@
 <template>
 	<aside>
 		<ul>
-			<li v-for="h in headers" :key="h.id">
+			<!-- <li v-for="h in headers" :key="h.id">
 				<nuxt-link :to="h.href">
 					{{ h.title }}
 				</nuxt-link>
+			</li> -->
+			<!-- The above <li> below are hard coded to show an example of how styles will look. This will be replaced by the commented out html code above. -->
+			<li>
+				<nuxt-link to="#title1">
+					title1
+				</nuxt-link>
+			</li>
+			<li>
+				<nuxt-link to="#title2">
+					title2
+				</nuxt-link>
+			</li>
+			<li>
+				<nuxt-link to="#title3">
+					title3
+				</nuxt-link>
 			</li>
 		</ul>
```

Comment:
I think this should all be placed inside a `<nav>` element. @lliskovoi shared a [link](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role) in the fluid-design channel that discusses using the `<nav>` element and in particular when there are multiple `<nav>` elements on the page.